tests/__init__.py

```diff
@@ -225,10 +224,6 @@ def execute_setup_py(project_dir, setup_args, disable_languages_test=False):
         del os.environ["_PYTHON_HOST_PLATFORM"]
 
     with push_dir(str(project_dir)), push_argv(["setup.py", *setup_args]), prepend_sys_path([str(project_dir)]):
-        # Restore master working set that is reset following call to "python setup.py test"
-        # See function "project_on_sys_path()" in setuptools.command.test
-        pkg_resources._initialize_master_working_set()  # type: ignore[attr-defined]
```

Comment:
Glad to see this one go :rocket:  ... if i recall it was not easy to find out at that time.